dsc/item_submission.py

```diff
@@ -66,6 +66,9 @@ class ItemSubmission:
     bitstream_s3_uris: list[str] | None = None
     metadata_s3_uri: str = ""
 
+    def asdict_subset(self, fields: list[str]) -> dict:
```

Comment:
Brief docstring about why this subset is selected (i.e. reporting) would be helpful